Components/Transforms/TranslationStackTransform/itkAdvancedTranslationTransform.h

```diff
@@ -0,0 +1,332 @@
+/*=========================================================================
+ *
+ *  Copyright UMC Utrecht and contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+/*=========================================================================
+
+  Program:   Insight Segmentation & Registration Toolkit
+  Module:    $RCSfile: itkAdvancedTranslationTransform.h,v $
```

Comment:
This file "Components/Transforms/TranslationStackTransform/itkAdvancedTranslationTransform.h" was removed with pull request https://github.com/SuperElastix/elastix/pull/447 commit 7fdca9705a29541590e22bde7d8234536e5f5d20  Can you please keep it removed?

Reply:
Thanks, I see you addressed my comment on itkAdvancedTranslationTransform.h 👍